django_walletpass/models.py

```diff
@@ -208,6 +208,8 @@ def write_to_model(self, instance=None):
         else:
             filename = f"{uuid.uuid1()}.pkpass"
 
+        if self.builded_pass_content is None:
+            raise ValueError(_("Cannot not save to model: builded_pass_content is None."))
```

Comment:
It can be either `Cannot save` or `Can't save`. Can you change it, please?

Reply:
sure, copied `Cannot` from the prior occurrence